twisted/conch/ssh/factory.py

```diff
@@ -58,7 +58,7 @@ def buildProtocol(self, addr):
                     'because we cannot find moduli file')
             t.supportedKeyExchanges = [
                 kexAlgorithm for kexAlgorithm in t.supportedKeyExchanges
-                if _kex.isFixedGroup(kexAlgorithm)]
+                if _kex.isFixedGroup(kexAlgorithm) or (hasattr(_kex, "isEllipticCurve") and _kex.isEllipticCurve(kexAlgorithm))]
```

Comment:
At one point it was otherwise an exception was thrown.

Whatever was causing it seems to have fixed itself.